rules/manager.go

```diff
@@ -579,3 +581,18 @@ func FromMaps(maps ...map[string]string) labels.Labels {
 
 	return labels.FromMap(mLables)
 }
+
+func ParseFile(fn string) (interface{}, error) {
+	content, err := os.ReadFile(fn)
+	if err != nil {
+		return nil, err
+	}
+
+	var rules interface{}
+	err = yaml.Unmarshal(content, &rules)
+	if err != nil {
+		return nil, err
+	}
+
+	return rules, nil
+}
```

Comment:
There's an existing function in the `rulefmt` package in prometheus called `ParseFile()` that seems to do both yaml validation as well as validation of the literal rule groups. Is this new function necessary? 

https://github.com/prometheus/prometheus/blob/9659e30dec7073703fb8548e7b0ad80dd0df48f0/model/rulefmt/rulefmt.go#L364-L375